### PR TITLE
WIP Presentation suggestion Kubernetes (specific topic TBD)

### PR DESCRIPTION
**Pull request by ChristopherGustafson**
# WIP Presentation suggestion Kubernetes (specific topic TBD)

## Members

Axel Pettersson (axp@kth.se)
GitHub: [Ackuq](https://github.com/Ackuq)

Christopher Gustafson (chrigu@kth.se)
Github: [ChristopherGustafson](https://github.com/ChristopherGustafson)

## Proposal

We would like to make a presentation on some part of Kubernetes. The specific part of Kubernetes we would like to cover is to be determined.

### CODE REVIEW

File: contributions/presentation/week2/axp-chrigu/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+Axel Pettersson (axp@kth.se)
+GitHub: [Ackuq](https://github.com/Ackuq)
+
+Christopher Gustafson (chrigu@kth.se)
+Github: [ChristopherGustafson](https://github.com/ChristopherGustafson)
+
+## Proposal
+
+We would like to make a presentation on some part of Kubernetes. The specific part of Kubernetes we would like to cover is to be determined.
```
Comment (cesarsotovalero) [inline contributions/presentation/week2/axp-chrigu/README.md:13]: Please add exactly the part of k8 that will be the focus of your presentation.

**Review by cesarsotovalero — COMMENTED** [review]
more information needs to be added.

**Comment by ChristopherGustafson** [comment]
We decided to wait with the presentation until a later week to focus on our other task. We will open a new PR when we decide to have it. :)

**Comment by cesarsotovalero** [comment]
> We will open a new PR when we decide to have it. :)

@ChristopherGustafson Looking forward to your new proposal.

**Pull request closed without merging**